src/shadowbox/model/access_key.ts

```diff
@@ -53,6 +53,8 @@ export interface AccessKeyRepository {
   createNewAccessKey(): Promise<AccessKey>;
   // Removes the access key given its id. Throws on failure.
   removeAccessKey(id: AccessKeyId);
+  // Returns the access key with the given id. Throws on failure.
+  getAccessKey(id: AccessKeyId): AccessKey;
```

Comment:
This is a reasonable addition to the API, and your code changes look solid. Thanks for carefully putting this together.

Could you clarify why you need this addition? Usually the listAccessKeys() is enough, so I'd like to better understand what the underlying need is.

Reply:
@fortuna 

Thanks for feedback.
I think this addition need for developing a 3rd-party API, for example I made a [npm package](https://github.com/murka/outlinevpn-api) to interact with outline server, but I must be use array filtering for look up user.